README.md

```diff
@@ -1,19 +1,32 @@
-### Hello there :suspect:, I am Paul Artushkov (also known as pavTiger)
-
-Since the early childhood, I've been interested in technology. I started with soldering some boards and Arduino components. 
+## Hello there :suspect:, I am Paul Artushkov (also known as pavTiger)
+Since early childhood, I've been interested in technology. I started with soldering some boards and Arduino components. 
 Then I slowly switched to Robotics olympiads with LEGO EV3 system. And went to RRO (Russian Robot Olympiad) in <b>Innopolis</b>. 
 As I went to the fifth grade, I switched to more practical programming (that was the time when I started using <b>GitHub</b>). 
 Enjoy hackathons very much and have been to many of <b>CROC<b> hacks (and won a few). Recently, I started coding Olympiad tasks in <b>Tinkoff Generation<b>.
 
 I study in an IT & Physics school named Silaeder, where we do many projects and hackathons.
 
-My biggest projects are:
-* Physarum modeling
-* TRON Legacy game
+Good-known languages: Python, C++, Javascript, Pascal
```

Comment:
I think "Good-known" is grammatically incorrect, the correct version is "Well-known", but it is kind of a synonym for famous. It means "languages that many people know", not "languages, which I know well"